app/stac_api/models.py

```diff
@@ -523,7 +532,12 @@ def upload_asset_to_path_hook(instance, filename=None):
         'Set uploaded file %s multihash %s to checksum:multihash; computation done in %ss',
```

Comment:
You may also change the `%ss` to `%.3fs` here as well

Reply:
done